.github/workflows/test_on_push.yml

```
@@ -7,25 +7,32 @@ on:
     branches:
     - main
 
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+env:
+  # The "FORCE_COLOR" variable, when set to 1,
+  # tells Nox to colorize itself.
+  FORCE_COLOR: "1"
```

This could be set to `3` to make this behaviour more aggressive, since some dependencies (like `click` back in the day) didn't/don't respect the behaviour fully:

```suggestion
env:
  # The "FORCE_COLOR" variable, when set to 1,
  # tells Nox to colorize itself.
  FORCE_COLOR: "3"
```

This will let tools make use of the full palette of 256 colours.

See: https://github.com/chalk/supports-color#api and https://force-color.org/